### PR TITLE
[HttpKernel] Add customization options to `#[MapQueryString]` and `#[MapRequestPayload]`

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Resolves https://github.com/symfony/symfony-docs/issues/18237
Resolves https://github.com/symfony/symfony-docs/issues/18229

### CODE REVIEW

File: controller.rst [hunk 1]
```diff
@@ -430,6 +430,16 @@ attribute in your controller::
         // ...
     }
 
+You can customize the validation groups used during the mapping thanks to the
+``validationGroups`` argument::
```
Comment (javiereguiluz) [inline controller.rst]: Super minor comment: should we call it `validationGroups argument` or `validationGroups option` ?

**Review by javiereguiluz — APPROVED** [review]
Thanks Alex!

**Review by javiereguiluz — APPROVED** [review]
Thanks Alex!

File: controller.rst (hunk 1 above)
Comment (alexandre-daubois) [reply controller.rst]: I personally prefer calling it `option` when it comes to attributes, even more when they're optional. That's updated accordingly!

**Comment by javiereguiluz** [comment]
Nice contribution. It's merged now. Thanks Alex!

**Pull request merged**